src/GTP.cpp

```diff
@@ -362,24 +372,59 @@ bool GTP::execute(GameState & game, std::string xinput) {
                 gtp_fail_printf(id, "syntax error");
                 return 1;
             }
+            if (analysis_output) {
+                cfg_analyze_interval_centis = interval;
+                if (id != -1) gtp_printf_raw("=%d\n", id);
```

Comment:
Isn't this the same as the regular gtp_printf calls? Except you're dealing with the id manually now, which is what those helper functions were for?

Reply:
The difference is that this is a multi-line response. The helper functions add \n\n at the end, and that is no good.